seller_data/admin.py

```
@@ -1,3 +1,26 @@
 from django.contrib import admin
+from .models import *
```

Ruff (0.11.9)

`from .models import *` used; unable to detect undefined names

(F403)